packages/gatsby-source-filesystem/src/create-remote-file-node.js

```diff
@@ -225,8 +225,11 @@ async function processRemoteNode({
 
   // Fetch the file.
   const response = await requestRemoteNode(url, headers, tmpFilename)
-  // Save the response headers for future requests.
-  await cache.set(cacheId(url), response.headers)
+
+  if (response.statusCode == 200) {
```

Comment:
Do we need to handle any other successful response status codes? Like `201` etc

I noticed that we also check for only `200` later in the file when moving temp data over so I'd guess we should be okay.